src/protocol/protocol.yml

```diff
@@ -373,6 +373,7 @@ BrowserType:
           - no-preference
         acceptDownloads: boolean?
         _traceDir: string?
+        _debugName: string?
```

Comment:
Underscore is good. We'll make it public once we are sure about the api. No need for docs yet.